stellargraph/random.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+`stellargraph.random` contains functions to control the randomness behaviour in StellarGraph.
```

Comment:
I think Sphinx uses reStructuredText not markdown, so this inline code http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup needs to be:

```suggestion
``stellargraph.random`` contains functions to control the randomness behaviour in StellarGraph.
```

- [ ] similarly for `` `tensorflow.random.set_seed` `` in `set_seed`.